1-exercises/B-setting-values/exercise2.js

```diff
@@ -16,7 +16,7 @@ let student = {
 */
 
 // write code here
-
+student.attendance =90;
```

Comment:
This works of course, but the task asked us to use bracket notation to achieve the result here.